Modules/_pickle.c

```diff
@@ -6233,7 +6233,7 @@ load_build(UnpicklerObject *self)
         slotstate = NULL;
 
     /* Set inst.__dict__ from the state dict (if any). */
-    if (state != Py_None) {
+    if (PyObject_IsTrue(state)) {
```

Comment:
Any reason you're changing this?

Reply:
Consistency with `pickle.py`.
But this PR documents an implementation detail and makes a quasi-non significant change, so feel free to close it.

Reply:
If you say so :-) I'm closing now.